app/src/viewmodels/camera/CameraViewModel.tsx

```diff
@@ -25,15 +25,30 @@ export default function useCameraViewModel(
   const camera = useRef<CameraView>(null);
   const [picture, setPicture] = useState<CameraCapturedPicture>();
   const [isCameraEnabled, setIsCameraEnabled] = useState(true);
-  const [flashMode, setFlashMode] = useState<FlashMode | string>("off");
+  const [flashMode, setFlashMode] = useState<FlashMode>("off");
   const [isFlashEnabled, setIsFlashEnabled] = useState(false);
   const [zoom, setZoom] = useState(0);
-  const [lastZoom, setLastZoom] = useState(0);
+  const [lastZoom] = useState(0);
```

Comment:
Why removing the set ?

Reply:
Well it wasn't used by the code so I removed it as it was flagged as an issue by SonarCloud, also if we ever need to use it we'll just write it back. The goal of this PR is to have a master branch with the minimum amount of issues such that we code on a clean base.